docs/experimentalists/pooler/grid/quickstart.md

```diff
@@ -10,5 +10,5 @@ You will need:
 you can import the grid pooler via:
 
 ```python
-from autora.experimentalist.pooler.grid import grid_pool
+from autora.experimentalist.grid_ import grid_pool
```

Comment:
Why is it `autora.experimentalist.grid_` and not just `autora.experimentalist.grid`?

Reply:
The problem here is tricky. When you run `pytest`, it imports any submodules using their unqualified name – it uses `grid` rather than `autora.experimentalist.grid`; `grid` rather than `autora.experimentalist pooler.grid`. 

Unfortunately, this means there are two things with the same name, and that throws an error.

For backwards compatibility, where we didn't want to break the old imports from `autora.experimentalist pooler.grid`, we want to leave that old module where it is. We have to have a different name, and the recommended way to do that is to add an underscore on the end.